helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/ClusterAccessor.java

```diff
@@ -671,4 +559,85 @@ private boolean doesClusterExist(String cluster) {
     }
     return history;
   }
+
+  private Response updateConfig(String clusterId, String commandStr, String content,
```

Comment:
That is a good refactoring. Just a nit suggestion, maybe we can move this to a Util class and apply to all other configs operations for different REST accessors such as ResourceConfig, InstanceConfig and so on.